wordpress-updatable/versions/0.1.2/schema.yaml

```diff
@@ -11,9 +11,6 @@ x-google-marketplace:
     - Feature
     recommended: true
 
-  managedUpdates:
-    kalmSupported: true
-
```

Comment:
Hm - each release is intended to be immutable, so we should avoid changing 0.1.2

Reply:
Ahh I made this change because kalm was causing it to fail for older versions, will add back